### PR TITLE
Add OpenSim.app v0.0.6

### PR DESCRIPTION
**Pull request by ronanrodrigo**
### Checklist
- [x] The commit message includes the cask’s name and version.
- [x] `brew cask audit --download opensim` is error-free.
- [x] `brew cask style --fix opensim` left no offenses.

Additionally, when **adding a new cask**:
- [x] Checked there are no open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there are no closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install opensim` worked successfully.
- [x] `brew cask uninstall opensim` worked successfully.

### CODE REVIEW

File: Casks/opensim.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'opensim' do
+  version '0.0.6'
+  sha256 '79cc8a041f470da4f463e8d0087d928dce9c2519b5ed5a6e221d47a205bc2a42'
+
+  url "https://github.com/luosheng/OpenSim/releases/download/#{version}/OpenSim.app.zip"
+  name 'OpenSim'
+  homepage 'https://github.com/luosheng/OpenSim'
```
Comment (vitorgalvao) [inline Casks/opensim.rb]: `https://github.com/luosheng/OpenSim/` (add trailing `/`).

File: Casks/opensim.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+cask 'opensim' do
+  version '0.0.6'
+  sha256 '79cc8a041f470da4f463e8d0087d928dce9c2519b5ed5a6e221d47a205bc2a42'
+
+  url "https://github.com/luosheng/OpenSim/releases/download/#{version}/OpenSim.app.zip"
```
Comment (vitorgalvao) [inline Casks/opensim.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

File: Casks/opensim.rb (hunk 2 above)
Comment (ronanrodrigo) [inline Casks/opensim.rb]: Done :)

File: Casks/opensim.rb (hunk 1 above)
Comment (ronanrodrigo) [inline Casks/opensim.rb]: Done

**Pull request merged**